.env.example

```diff
@@ -24,3 +24,7 @@ DOCKER_COMPOSE_SERVER_URL=https://condo.dev.doma.ai
 
 # feature flags configuration
 FEATURE_FLAGS_CONFIG='["billing"]'
+
+# FIREBASE_CONFIG_JSON contents can be retrieved from https://console.firebase.google.com/project/<PROJECT_ID>/settings/serviceaccounts/adminsdk
+FIREBASE_CONFIG_JSON='{"type": "service_account", "project_id": "<ROJECT_ID>", "private_key_id": "<PRIVATE_KEY_ID", "private_key": "<PRIVATE_KEY>", "client_email": "<FIREBASE_ADMIN_SDK_EMAIL> at .gserviceaccount.com", "client_id": "<CLIENT_ID", "auth_uri": "https://accounts.google.com/o/oauth2/auth", "token_uri": "https://oauth2.googleapis.com/token", "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs", "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/<CLIENT_EMAIL_VALUE>"}'
```

Comment:
This file is really used in CI tests. You need to comment this lines if you want to add it as an exmaple

Reply:
fixed